### ISSUE TITLE
oEmbed

### ISSUE THREAD

**Issue by mitchelljkotler** [MuckRock/muckrock#1568]

**Comment by TJKenney** [comment]
How tough would this be to do?

**Comment by mitchelljkotler** [comment]
I don't think it would be too hard, but it is not trivial.  It is also not a top priority at the moment.

**Comment by knowtheory** [comment]
@mitchelljkotler do you have a sense of what things folks want to be able to embed?

**Comment by mitchelljkotler** [comment]
Most importantly would be crowd funding campaigns (ie https://www.muckrock.com/project/subjects-matter-fbi-files-10/) and assignments.  Both of these have embed codes, which are just snippets of html to insert an iframe.  So we would just need to expose those snippets according to the oEmbed spec.

We also give out embed codes for files, but this is actually just a wrapper around the document cloud embed, so we should probably rethink that.

There are also datasets, which have an embed code, but are fairly underdeveloped and haven't been used extensively for anything yet - we'd probably add these in later.

**Comment by TJKenney** [comment]
@mitchelljkotler If there is anything I could do to help, please let me know.
